config/locales/codelists/2_03/iati.en.yml

```diff
@@ -658,6 +658,18 @@ en:
       '920': SUPPORT TO NON- GOVERNMENTAL ORGANISATIONS (NGOs)
       '930': Refugees in Donor Countries
       '998': Unallocated / Unspecified
+    programme_status:
```

Comment:
True. I didn't notice. What would be a good place for it? `vendor/data/codelists/BEIS` ?

Reply:
I'm finally leaving this where it is. This is to avoid losing the logic that allows a user to see the locale value for `programme_status` when they see the activity details